scripts/Inelastic/CrystalField/fitting.py

```diff
@@ -1289,6 +1292,112 @@ def monte_carlo(self, **kwargs):
             self._monte_carlo_single(**kwargs)
         self.model.FixAllPeaks = fix_all_peaks
 
+    def two_step_fit(self, OverwriteMaxIterations: list = None, OverwriteMinimizers: list = None, Iterations: int = 20) -> None:
+        from mantid.kernel import logger
```

Comment:
Imports should be at the top of a file

Reply:
There's several places we import mid-method which need tweaking, so I've left the others uncommented to avoid noise